### PR TITLE
Add config for maintenance mode, Skip write to DB and flushing spool …

### PR DESCRIPTION
**Pull request by norkans7**
…and write to spool only

### CODE REVIEW

**Comment by ericnewcomer** [comment]
Why no redis? Seems duplicate checking on spooling is still valuable no?

File: backends/rapidpro/msg.go [hunk 1]
```diff
@@ -72,14 +73,19 @@ func writeMsg(ctx context.Context, b *backend, msg courier.Msg) error {
...
 
 	// try to write it our db
-	err := writeMsgToDB(ctx, b, m)
+	if !b.config.Maintenance {
+		err = writeMsgToDB(ctx, b, m)
+	}
 
-	// fail? spool for later
+	// fail? log
 	if err != nil {
```
Comment (ericnewcomer) [inline backends/rapidpro/msg.go]: what err is this checking?

**Comment by norkans7** [comment]
I thought that is not useful since the UUID are nil in that case also we use the current timestamp for spool files so I thought the seen should written only when not in maintenance mode

**Comment by norkans7** [comment]
But I guess we can make redis available for the external ID seen only

**Pull request merged**